File: Code/java/vowel.java
import java.util.Scanner;

class Vowel
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Alphabet: ");
        String alphabet =  scanner.next().toUpperCase();
        if ("AEIOU".contains(alphabet))
            System.out.println(alphabet + " is a Vowel");
        else
            System.out.println(alphabet + " is a Consonant");

        scanner.close();
    }
}

File: Code/cpp/vowel.cpp
#include<iostream>
using namespace std;
int main()
{
    char alphabet, vowels[5] = {'A', 'E', 'I', 'O', 'U'};
    string result = "Consonant";
    int i;

    cout << "Enter Alphabet: ";
    cin >> alphabet;

    alphabet = toupper(alphabet);
    for (i = 0; i < 5; i++)
    {
        if (alphabet == vowels[i])
        {
            result.replace(0, 10,"Vowel");
            break;
        }
    }
    cout << alphabet << " is a " << result;
    return 0;
}